src/ui/components/SecondaryToolbox/NewConsole.tsx

```diff
@@ -85,9 +83,9 @@ function JSTermWrapper() {
     addMessage({
       expression,
       frameId: frameId || null,
-      pauseId,
-      point,
-      time,
+      pauseId: pause.pauseId,
+      point: pause.point!,
+      time: pause.time!,
```

Comment:
Is it safe to assume we'll always have a `point` and `time` here?

Reply:
Yes, those are only set to `null` briefly during the initialisation of the Pause, but by the time the Pause is added to `pausesById` (which is where we get this Pause from), they are always updated.